routes/invitation.js

```diff
@@ -2,9 +2,10 @@ const express = require('express');
 const router = express.Router();
 const Invitation = require('../models/Invitation');
 const User = require('../models/User');
+const auth = require('../middleware/auth');
 
-//PUT request to approve invitation
-router.put('/:id/approve', async (req, res) => {
+//PUT request to approve invitation PRIVATE ROUTE
+router.put('/:id/approve', auth, async (req, res) => {
```

Comment:
We also need to add a check that the user who is authenticated is the user whose invitation will be approved, and similar for all protected routes